clarifai/cli/model.py

```diff
@@ -39,17 +39,38 @@ def upload(model_path, download_checkpoints, skip_dockerfile):
     type=click.Path(exists=True),
     required=True,
     help='Path to the model directory.')
+@click.option(
+    '--mode',
+    type=click.Choice(['env', 'container'], case_sensitive=False),
+    default='env',
+    show_default=True,
+    help=
+    'Specify how to test the model locally: "env" for virtual environment or "container" for Docker container. Defaults to "env".'
+)
 @click.option(
     '--keep_env',
     is_flag=True,
-    help='Flag to keep the virtual environment after testing the model\
-    locally. Defaults to False, which will delete the virtual environment after testing.')
-def test_locally(model_path, keep_env=False):
+    help=
+    'Keep the virtual environment after testing the model locally (applicable for virtualenv mode). Defaults to False.'
+)
+@click.option(
+    '--keep_image',
+    is_flag=True,
+    help=
+    'Keep the Docker image after testing the model locally (applicable for container mode). Defaults to False.'
+)
+def test_locally(model_path, keep_env=False, keep_image=False, mode='env'):
   """Test model locally."""
```

Comment:
Can we add this for clarity?
```suggestion
  """Test model locally."""
if mode == 'env' and keep_image; raise ValueError("'keep_image' is applicable only for 'container' mode")
if mode == 'container' and keep_env; raise ValueError("'keep_env' is applicable only for 'env' mode")
```

Reply:
updated it!